internal/protos/orchestrator_service.pb.go

```diff
@@ -91,13 +91,62 @@ func (OrchestrationStatus) EnumDescriptor() ([]byte, []int) {
 	return file_orchestrator_service_proto_rawDescGZIP(), []int{0}
 }
 
+type CreateOrchestrationAction int32
+
+const (
+	CreateOrchestrationAction_ERROR     CreateOrchestrationAction = 0
+	CreateOrchestrationAction_SKIP      CreateOrchestrationAction = 1
```

Comment:
Should we use IGNORE instead of SKIP? I feel like IGNORE might be more semantically clear.